woocommerce/woocommerce
# [Experimental] Product Filter: Attribute (Beta) block - Fix block preview

### Changes proposed in this Pull Request:

This PR implements the logic needed to make the block preview appear correctly for the Product Filter: Attribute (Beta) block when hovering over it in the block inserter.





Closes #45491 .



### How to test the changes in this Pull Request:



1. Navigate to Single Product template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an "Add Block" option. Click on it.
4. In the block inserter, search for "Product Filter" or browse the available blocks until you find them.
5. Hover the mouse over the "Product Filter: Attribute (Beta)" block and ensure the block preview is correctly rendered.

| Before | After |
|--------|--------|
| ![CleanShot 2024-03-13 at 18 06 43@2x](https://github.com/woocommerce/woocommerce/assets/20469356/8baf4039-a47b-4d83-a754-6a38e89ed3e1) | ![CleanShot 2024-03-13 at 18 06 43@2x](https://github.com/woocommerce/woocommerce/assets/20469356/1d3d910a-36f0-4d51-a52d-5420974a95e0) | 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Add block preview to Product Filter: Attribute (Beta) block

#### Comment 

</details>
